torch/_tensor_docs.py

```diff
@@ -3202,10 +3238,39 @@ def callable(a, b) -> number
                r"""
 sparse_dim() -> int
 
-If :attr:`self` is a sparse COO tensor (i.e., with ``torch.sparse_coo`` layout),
+If :attr:`self` is a :ref:`sparse tensor <sparse-docs>`,
```

Comment:
I'm noticing a lot of both "sparse COO tensor" and "sparse tensor" appearing in the documentation. What's the logic for when one is used vs. another?

Reply:
Flipping the sentence around may clarify what this function does. 

"Returns the number of sparse dimensions in self when self is a sparse tensor. Throws an error if self is not a sparse tensor."

Reply:
Yes, the usage of these terms is intentional: "sparse tensor" is a common denominator to "sparse COO tensor" and the future "sparse GCS tensor" (see https://github.com/pytorch/pytorch/pull/44190), and sometimes we need to distinguish between the sparse storage formats, for instance, the concept of uncoalesced/coalesced tensors can be used only for sparse COO tensors.

Reply:
Fixed via sentence flip.